src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -107,6 +107,16 @@
     private transient boolean closed;
 
     /**
+     * Maximum number of log lines for a stack trace
+     */
+    public static final int DEFUALT_MAX_LENGTH = 1000;
```

Comment:
@dean-e-clark `public static final` should not be used at all: http://www.yegor256.com/2015/07/06/public-static-literals.html